src/pages/pattern-library.tsx

```diff
@@ -145,6 +147,19 @@ function Page() {
               </li>
             </ul>
           </section>
+
+          <section className="grid-section grid-content">
+            <h2 className="title-subsection">CartToggle</h2>
+            <UIProvider>
+              <CartProvider>
+                <ul className="list-vertical">
+                  <li>
+                    <CartToggle />
+                  </li>
+                </ul>
+              </CartProvider>
```

Comment:
This is not a blocker but do we need these SDK providers at the pattern library? Maybe it's better to use the component without integrations here.

Reply:
The component  use contexts, then we need to put their inside a provider

Reply:
I got it. My point is regarding using a real context in the pattern library. If it's not too much work, mocking it would be better.